extras/ffmpeg-decoder/src/ffmpeg/avcodec.rs

```diff
@@ -468,6 +469,10 @@ impl Iterator for AvPixelFormatIterator {
 /// A codec context from which decoding can be performed.
 pub struct AvCodecContext(*mut ffi::AVCodecContext);
 
+/// SAFETY: safe because mutable pointers are exclusive en valid (implicit lifetime annotation).
+unsafe impl Send for AvCodecContext {}
+unsafe impl Sync for AvCodecContext {}
```

Comment:
I think you can say `Sync` is safe on this one because it does not allow mutable aliasing.